plugins/lookup/tss.py

```diff
@@ -118,15 +118,17 @@
 sdk_is_missing = False
 
 try:
-    from thycotic import __version__ as sdk_version
-    from thycotic.secrets.server import (
-        SecretServer,
-        SecretServerError,
-        PasswordGrantAuthorizer,
-    )
+    from thycotic.secrets.server import SecretServer, SecretServerError
```

Comment:
```suggestion
    import thycotic.secrets.server as server
```
This way there's no need for the conditional import in `LookupModule.Client`.

Reply:
Why not just `from thycotic.secrets import server`?

Reply:
I can't say I'm a huge fan of this option as that will import the whole SDK at runtime and not just the 3 classes that are used. While the current SDK isn't _that_ big. We are currently road mapping and developing the SDK, and it will grow much bigger in the not too distant future.

This is probably subjective, but I feel it'll also take away readability as I'll have to pre-pend `server` to all class methods.

Reply:
- It's not possible to import members independently of the module they are defined in so there is no difference in terms of what is loaded into the cache. (from foo import bar just adds bar unqualified to the module namespace)
- It's pretty common to import third-party packages with an alias to keep the namespace clean. (For example `np` or `pd`; I meant to suggest `tss` in my original comment)
- Regardless of how the classes are imported here importing within a function is not good practice so the conditional import of `PasswordGrantAuthorizer` should move into the module scope either way.